src/DrugstoreApp/Backend/DrugstoreAPI/Drugstore/Repository/Interfaces/IHospitalRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Drugstore.Models;
+
+namespace Drugstore.Repository.Interfaces
+{
+    public interface IHospitalRepository
+    {
+        public List<Hospital> GetAll();
+        public bool GetHospitalById(int id);
+        public int GetIdByName(string name);
+        public string GetKeyByName(string name);
+        public bool Save(Hospital newHospital);
```

Comment:
Also Save should return object created in db.

Reply:
I will use Command-Query separation pattern, so it's gonna be void!